lte/gateway/python/magma/pipelined/app/policy_mixin.py

```diff
@@ -36,6 +36,8 @@
 
 PROCESS_STATS = 0x0
 IGNORE_STATS = 0x1
+DROP_RULE_STATS = 0x2
+
```

Comment:
since drop is default action, can we assign DROP_RULE_STATS zero value. This way there is no need to set this it in the scratch register.
For all UE session there would be single flow in enforcement(main_table) table which resubmit it to stats table.
In stats table we could have low priority flow that match on specific rule to get stats.

Reply:
sure sounds good, I can refactor